sdv/sequential/par.py

```diff
@@ -423,11 +423,16 @@ def _fit_sequence_columns(self, timeseries_data):
             kind = dtype.kind
             if kind in ('i', 'f'):
                 data_type = 'continuous'
-                # Check if metadata overrides this data type
-                if self.metadata.columns.get(field, {}).get('sdtype', None) == 'categorical':
-                    data_type = 'categorical'
+                sdtype = self.metadata.columns.get(field, {}).get('sdtype', None)
```

Comment:
Should we raise an issue to update `Par` to use the new unified metadata ?

Reply:
@amontanez24 What do you think?